src/store/modules/config.ts

```diff
@@ -2,7 +2,6 @@ import { randomHex } from '@/modules/encryption/crypto';
 import {
   AccountToScrapeConfig, Config, OutputVendorsConfig, OutputVendorsNames
 } from '@/originalBudgetTrackingApp/configManager/configManager';
-import defaultConfig from '@/originalBudgetTrackingApp/configManager/defaultConfig';
```

Comment:
So you don't have problems like 'cannot access to "something" of undefined' when you just start the application without saved config?

Reply:
The `configManager.getConfig()` function has a fallback to the default config if there is no config file defined, so if the store already fetched the config from the config manager there should be no problem.

Since you have the splash screen until the store is loaded I think we are good.